content/app-management/kubernetes-serverless-without-the-vendor-lock-in-heres-how/_index.md

```
@@ -0,0 +1,883 @@
+
+There's a new resource in the mix. The Composition created a KEDA `ScaledObject` that watches the request rate coming through the gateway and adjusts replicas between 1 and 5.
+
+Let's look at the destination cluster to see what's there now.
+
+```sh
+kubectl --kubeconfig kubeconfig-remote.yaml --namespace dev get all,httproutes,scaledobjects
+```
+
+Here's what that gives us.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)